python/pyspark/testing/__init__.py

```diff
@@ -14,3 +14,5 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+
+__all__ = ["assertSparkDFEquality", "assertDFEquality"]
```

Comment:
Can we borrow how unittest and/or pandas testing util modules name them? e.g., unittest one is `assertEqual`.